tests/test_async_patterns.py

```
@@ -56,35 +56,51 @@ async def test_coroutine():
             """
             Asynchronously tests the Matrix connection coroutine, ensuring proper awaiting and client initialization.
 
-            Simulates the Matrix connection process by mocking SSL context, certificate path, and Matrix AsyncClient. Verifies that the `connect_matrix` function awaits the sync method and returns a valid client instance.
+            Simulates the Matrix connection process by mocking SSL context, certificate path, and Matrix AsyncClient. Verifies that `connect_matrix` function awaits the sync method and returns a valid client instance.
             """
+            # Mock credentials loading to prevent using real credentials
+            mock_auth_info = MagicMock()
+            mock_auth_info.homeserver = "https://matrix.org"
+            mock_auth_info.access_token = "test_token"
```

Ruff (0.14.14)

[error] Possible hardcoded password assigned to: "access_token"

(S105)